### PR TITLE
FRONT-6932 update client version in readme

### PR DESCRIPTION
**Pull request by stuart-gill**

### CODE REVIEW

**Comment by stuart-gill** [comment]
One question: I'm not sure why the <tag> in pom.xml changed to 6.0.27 from 6.0.29 when the new version was released. Does that need to be changed back manually? [Here is that commit](https://github.com/scalyr/Scalyr-Java-Client/commit/23fddea1704c1340cc1466e3771122da51a9a718).

File: readme.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Add the following dependency to your project's pom.xml (check [Maven Central](ht
         <dependency>
             <groupId>com.scalyr</groupId>
             <artifactId>scalyr-client</artifactId>
-            <version>6.0.27</version>
+            <version>6.0.29</version>
```
Comment (djungyvr) [inline readme.md:23]: Nice! Looks like line 29 also needs updating.

**Review by djungyvr — COMMENTED** [review]
> One question: I'm not sure why the in pom.xml changed to 6.0.27 from 6.0.29 when the new version was released. Does that need to be changed back manually? [Here is that commit](https://github.com/scalyr/Scalyr-Java-Client/commit/23fddea1704c1340cc1466e3771122da51a9a718).

My guess is that when we prepare for a release it will set the tag correctly and then once it's done it will set it to what it was before.

**Comment by stuart-gill** [comment]
Good catch! I changed the link text as well, to differentiate from the other link to Maven Central.

**Review by djungyvr — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**